eth_account/account.py

```diff
@@ -191,7 +192,11 @@ def encrypt(cls, private_key, password, kdf=None):
 
         password_bytes = text_if_str(to_bytes, password)
         assert len(key_bytes) == 32
-        return create_keyfile_json(key_bytes, password_bytes, kdf=kdf)
+
+        if iterations is None:
+            return create_keyfile_json(key_bytes, password_bytes, kdf=kdf)
+        else:
+            return create_keyfile_json(key_bytes, password_bytes, kdf=kdf, iterations=iterations)
```

Comment:
I decided to pass `iterations` to the method as-is (vs. another conditional block to not pass this if `iterations` is `None`) since [`create_keyfile_json`](https://github.com/ethereum/eth-keyfile/blob/v0.5.1/eth_keyfile/keyfile.py#L39) calls `_create_v3_keyfile_json` which [has a check if this is `None`](https://github.com/ethereum/eth-keyfile/blob/v0.5.1/eth_keyfile/keyfile.py#L87-L88)]. If we want more safety in case that goes away since that's an internal check in a private method, we can conditionally pass it as well similarly to how we do with the KDF.

Reply:
What you say make sense here, perhaps it would be even better to have it as a code comment rather than a PR comment. That way would be IMO more future proof.
Also you may want to enforce/unittest this behaviour via `unittest.mock`  (checking actual mock call) to make sure this feature is covered

Reply:
Sure, I'll have to look into how to test that since the method is in `eth_keyfile` and not the `Account` object.

Reply:
I *think* I'm :-1: on this precaution.  I understand the reasoning but given that the `create_keyfile_json` API is public we'd issue a major version bump for any breaking API change.

Reply:
So you want me to just pass the iterations to the param regardless of if it's `None`?

Reply:
Yes, directly passing `None` in should be equivalent.

Reply:
Fixed! 65864f3